src/App.js

```diff
@@ -0,0 +1,76 @@
+import './App.css';
+
+function App() {
+  const name = "Cindy";
+  var nameComponent = (name === "Cindy")?
+  <h1 className="darkpurple-text"> My name is {name}!</h1> : <p> My name is not Cindy.</p>
+  const buttonFunction = () =>
+    console.log("The button was clicked.");
+    <button onClick="window.open='https://www.youtube.com/watch?v=rEq1Z0bjdwc';"></button>
+  
+  const onChangeFunction = (event) => {
+    console.log(event.target.value);
+  }
+
+  return (
+    <div className="App-header">
+      <header className = "App-header">
+        <div>{nameComponent}</div>
+        <button onClick={buttonFunction}>Click Me</button>
+      </header>
+
+      <a href="https://www.youtube.com/watch?v=rEq1Z0bjdwc" className="App-header">
+        If the button doesn't take you to the link, click here instead!</a>
+
+      <p className="App-header">What is your name?</p>
+
+      <header className="App-header">
+        <input type="text" onChange={onChangeFunction}/>
+      </header>
+
+      <p className="App-header">Nice to meet you!</p>
+
+      <p className="lilac-text">I am from Fargo, North Dakota.</p>
+      <img src="Fargo.jpg" widgth="500" height="250" alt="Fargo"></img>
+      <p className="lilac-text">This term, I'll be living with a few friends in NYC!</p>
+      <img src="NYC.jpg" width="445" height="250" alt="NYC"></img>
+        
+      <div className= "App-header">
+          <p className="lilac-text">I am planning on majoring in computer science and economics.</p>
+    
+          <p className="lilac-text">The classes I'm taking for this quarter are: </p>
+              <ul className="brown-text">
```

Comment:
a list like this could be a great time to use the `.map` function. for example you could do this instead:
`const classes = ["ECON 1", "PHYS 3", "WRIT 5", "WGSS 15"];`
 `const classMap = classes.map((class) => { return( <li>{class}</li> ) });`
and then you could pop `classMap` into your html by just having:
`<ul>{classMap}</ul>`
not necessary, but just an example of where mapping might come in handy to reduce repeated code :).